target_chains/ton/contracts/contracts/common/constants.fc

```diff
@@ -9,3 +9,5 @@ const int GUARDIAN_SET_EXPIRY = 86400; ;; 1 day in seconds
 const int UPGRADE_MODULE = 0x0000000000000000000000000000000000000000000000000000000000436f7265; ;; "Core" (left-padded to 256 bits) in hex
 
 const int WORMHOLE_MERKLE_UPDATE_TYPE = 0;
+
+const int WORKCHAIN = 0;
```

Comment:
What is this?

Reply:
there are 2 blockchains on TON: https://docs.ton.org/learn/overviews/ton-blockchain#blockchain
and they have different costs depending on the chains: https://docs.ton.org/learn/overviews/ton-blockchain#blockchain

will add comment to the file to make it clearer